File: FishBun/src/main/java/com/sangcomz/fishbun/util/SquareFrameLayout.java
package com.sangcomz.fishbun.util;

import android.content.Context;
import android.util.AttributeSet;

/**
 * Created by sangc on 2015-12-26.
 */
public class SquareFrameLayout extends android.support.v7.widget.ContentFrameLayout {
    public SquareFrameLayout(Context context) {
        super(context);
    }

    public SquareFrameLayout(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public SquareFrameLayout(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec);
    }
}

File: FishBun/src/main/java/com/sangcomz/fishbun/util/SquareFrameLayout.kt
package com.sangcomz.fishbun.util

import android.content.Context
import android.util.AttributeSet

/**
 * Created by sangc on 2015-12-26.
 */
class SquareFrameLayout @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyleAttr: Int = 0) : androidx.appcompat.widget.ContentFrameLayout(context, attrs, defStyleAttr) {

    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec)
    }
}
